components/product/ProductInfo.js

```diff
@@ -10,15 +13,19 @@ import SpecialNotice from './SpecialNotice';
 function ProductInfo({ product }) {
   const { name, id, category, points, customerCost } = product;
   return (
-    <View style={{ width: 300 }}>
+    <ProductInfoContainer>
       <Title>{name}</Title>
-      <Subtitle>Category: {category}</Subtitle>
-      <View style={{ flexDirection: 'row' }}>
-        <Subtitle>${customerCost} each</Subtitle>
-        <Subtitle>Points Earned: {points}</Subtitle>
-      </View>
-      <SpecialNotice />
-    </View>
+      <ProductInfoCaptionContainer>
+        <Caption>${customerCost} each</Caption>
```

Comment:
Should probably be rounded to the nearest two decimal places (i.e. for banana it's showing up as $0.5 right now)

Reply:
+1, try `toFixed(2)` !